src/components/CardStatus/CardStatus.js

```diff
@@ -1,48 +1,65 @@
-import React, {useState, useEffect} from 'react'
-import './CardStatus.css'
+import React, { useState, useEffect } from "react";
+import "./CardStatus.css";
 
-export  const CardStatus = (props) => {
+/**
+ * 
+ *  I will write in this component but it apply to a lot of components
+    In component like this one you set redoundant statuses and for that
+    you needed to write effect to set thenm. This force react to do 
+    un-necessary re-render of the inner elements. 
+    it's better to use just the props.
+    States and effect need to be use when you have external interactivity 
+    from the props. in this case is not make a lot of sense. 
+    If you need transformation maybe it's better to use a useMemo or a useCallback.  
+ */
+export const CardStatus = (props) => {
+  const { booked, bookable } = props;
 
-    const {booked, bookable} = props
+  const [status, setStatus] = useState("");
+  const [statusBackgroundColor, setStatusBackgroundColor] = useState("");
 
-    const [status, setStatus] = useState('')
-    const [statusBackgroundColor, setStatusBackgroundColor] = useState('')
+  /**
+   * write css rule inside javascript and use it in style prop is a bad
+   * practise because doesn't allow you to use all the good things about css
+   * in this case would be much better to define classes and use them instead of styles
+   * props.
+   */
```

Comment:
In this one you mean create an object with the classes for example, and applying that to the class of the jsx item? If that is accurate, would you say I should always describe css in js instead of creating a separate file for it?